### ISSUE TITLE
docs: e2e_processing_latency_percentiles settings

### ISSUE THREAD

**Issue by JakeXu** [nsqio/nsq#899]
From the github, I found the "ndsq.cfg.example", it wrote like this:
## message processing time percentiles to keep track of (float)
e2e_processing_latency_percentiles = [
    100.0,
    99.0,
    95.0
]

But, on the official document, Command Line Options describe:
-e2e-processing-latency-percentile value
    message processing time percentiles (as float (0, 1.0]) to track (can be specified multiple times or comma separated '1.0,0.99,0.95', default none)

Which is right? Tks

**Comment by ploxiln** [comment]
Both are correct. If you set this in the config file, you use an array. If you set this with a command-line flag, you use commas.

https://github.com/nsqio/nsq/blob/master/nsqd/options.go#L64
https://github.com/mreiferson/go-options/blob/master/options.go#L192

**Comment by jehiah** [comment]
@ploxiln I think the question here is are the values in the range `0, 1.0` or `0, 100`

**Comment by ploxiln** [comment]
Ah, that is a good question. I had to hunt all the way into https://github.com/bmizerany/perks/ to be sure. Each value needs to be in the range `(0,1.0)` (note that this is not inclusive of 1.0, there's a `/ (1 - q)` in bmizerany/perks).

I'm also not sure if the command-line flag actually works, it doesn't look like it copies the percentiles option to `opts`:

```go
	// End to end percentile flags
	e2eProcessingLatencyPercentiles := app.FloatArray{}
	flagSet.Var(&e2eProcessingLatencyPercentiles, "e2e-processing-latency-percentile", "message processing time percentiles (as float (0, 1.0]) to track (can be specified multiple times or comma separated '1.0,0.99,0.95', default none)")
	flagSet.Duration("e2e-processing-latency-window-time", opts.E2EProcessingLatencyWindowTime, "calculate end to end latency quantiles for this duration of time (ie: 60s would only show quantile calculations from the past 60 seconds)")
```

**Comment by ploxiln** [comment]
I take back that bit about not being sure the command-line flag works, go-options gets the reference from the flagSet Var.

Anyway, a documentation/examples update should be sufficient to resolve this issue.

**Comment by mreiferson** [comment]
fixed in #988
